src/System.Net.Primitives/tests/FunctionalTests/CookieTest/CookiePortTest.cs

```diff
@@ -1,81 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Net;
-
-using Xunit;
-
-namespace System.Net.Primitives.Functional.Tests
```

Comment:
What was this file deleted? We want to keep existing tests as they are, with small or no modifications to maintain parity with .Net Desktop.